newsletters/2023-03-newsletter.md

```diff
@@ -36,30 +36,33 @@ Hello! All Vipps customers that have API access now receive this technical
 newsletter.
 
 The newsletter is sent 5-10 times a year and contains important updates and
-information about our APIs: https://vippsas.github.io/vipps-developer-docs/docs/APIs
+information about our [APIs](https://vippsas.github.io/vipps-developer-docs/docs/APIs).
 
-Newsletter archive: https://vippsas.github.io/vipps-developer-docs/docs/vipps-developers/newsletters
+See: [Newsletter archive](https://vippsas.github.io/vipps-developer-docs/docs/vipps-developers/newsletters).
 
 (You can unsubscribe if you want - see the link at the bottom)
```

Comment:
There is no link at the bottom. How do they unsubscribe? By opening a GitHub issue or contacting support? If it's not easy, we should remove that sentence.